src/lib/annotations/Annotator.js

```diff
@@ -75,6 +77,13 @@ class Annotator extends EventEmitter {
             });
         }
 
+        // Destroy all annotate buttons
+        Object.keys(this.modeButtons).forEach((type) => {
```

Comment:
Just looked at the code in BaseViewer. Looks like this is fine since baseviewer passes in a constant object that always exists.